examples/EnableServiceProfilerForAzureFunction/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated6.0 AS base
+WORKDIR /home/site/wwwroot
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["EnableServiceProfilerForAzureFunction/SampleFunctionApp.csproj", "SampleFunctionApp/"]
+RUN dotnet restore "SampleFunctionApp/SampleFunctionApp.csproj"
+COPY . .
+WORKDIR "/src/SampleFunctionApp"
+RUN dotnet build "SampleFunctionApp.csproj" -c Release -o /app/build
```

Comment:
Is it redundant to perform a `dotnet build` before your `dotnet publish`?  We are not actually doing anything with the `/app/build` output after this line, right?